src/test/scala/chiselTests/EndToEndSMTBaseSpec.scala

```diff
@@ -0,0 +1,97 @@
+package chiselTests
+
+import chisel3.Module
+import chisel3.stage.{ChiselGeneratorAnnotation, ChiselStage}
+import firrtl.annotations.Annotation
+import firrtl.options.{OutputAnnotationFileAnnotation, TargetDirAnnotation}
+import firrtl.stage.OutputFileAnnotation
+import firrtl.util.BackendCompilationUtilities.timeStamp
+import logger.{LazyLogging, LogLevel, LogLevelAnnotation}
+import org.scalatest.flatspec.AnyFlatSpec
+import os._
+
+abstract class EndToEndSMTBaseSpec extends AnyFlatSpec {
```

Comment:
Please change this to `SMTModelCheckingSpec`.

Reply:
Please also add a comment that this is taken from the firrtl tests.

Reply:
Sure:)